thoth/storages/graph/postgres.py

```diff
@@ -5477,6 +5487,17 @@ def sync_solver_result(self, document: dict, *, force: bool = False) -> None:
                                 marker_evaluation_result=dependency.get("marker_evaluation_result", True),
                             )
 
+            # Sync imoprted package from result:tree except for test module, all the namespaced calls
+            # modules starting with __ also excluded
+            # For Thoth-solver-dataset-v2 ["result"]["tree"]["packages"] has only one module to import
+            import_package =[]
+            for package_call in document["result"]["tree"]["packages"]:
+                if package_call.count(".") == 0 && package_call[:4] != "test"
+                    && package_call[0] != "_":
```

Comment:
```suggestion
                    and package_call[0] != "_":
```

Reply:
Could you give examples for entries that are skipped?

Reply:
> Could you give examples for entries that are skipped?

Hi @fridex, thank you for helping!
Yes, when exploring [Thoth solver dataset v2 ](https://www.kaggle.com/thothstation/thoth-solver-dataset-v10?select=thoth-solver-dataset-v2.0) for report with documet_id = solver-rhel-8-py38-210713042150-50168537c03f93c5 tree:packages looks like:
```
['_plotly_future_',
 '_plotly_utils',
 'plotly',
 'plotlywidget',
 '_plotly_utils.colors',
 'plotly.colors',
 'plotly.data',
 'plotly.express',
 'plotly.figure_factory',
 'plotly.graph_objs',
 'plotly.io',
 'plotly.matplotlylib',
 'plotly.offline',
 'plotly.plotly',
 'plotly.validators',
 'plotly.graph_objs.area',
...
]
```

Here we would like to skip all hidden entry imports that start with '_'.
Also AFAIK it is the only one that has two modules, but `plotlywidget` is not on pypi.org.